ros_buildfarm/templates/snippet/plot_plugin.xml.em

```diff
@@ -3,6 +3,7 @@
 @[for plot_group, plot_list in plots.items()]@
 @[for plot in plot_list]@
         <hudson.plugins.plot.Plot>
+          <description>@("".join({ "&": "&amp;", '"': "&quot;", "'": "&apos;", ">": "&gt;", "<": "&lt;"}.get(c,c) for c in plot.description))</description>
```

Comment:
Is there a difference in behavior between a non-existent, null/self-closing, and empty tag? Do we end up with additional blank space in the plot where the description would be, even though a description wasn't specified?